onnx/test/cpp/function_context_test.cc

```diff
@@ -149,7 +149,265 @@ void RegisterCustomFunctionSchema() {
   (void)unused;
 }
 
-// Test for Context dependant function with type context
+void RegisterVersionedLogSoftmaxFunctionSchema(const std::string& op_type, bool with_missing_version) {
+  // SinceVersion of a function operator tells the last opset version where its semantic is defined.
+  // Function bodies and context dependent function builders also have versions.
+  // A function's version specifies its valid range since its since_version till the since_version
+  // of the next function of the same funtion op.
+  std::cout << "StepA\n"; 
```

Comment:
Delete?

Reply:
Sorry this is temporary for a mysterious CI failure.